packages/block-editor/src/components/link-control/style.scss

```diff
@@ -383,7 +392,7 @@ $preview-image-height: 140px;
 .block-editor-link-control__settings {
 	border-top: $border-width solid $gray-300;
 	margin: 0;
-	padding: $grid-unit-20 $grid-unit-30;
+	padding: 16px;
```

Comment:
I'd still use the grid units here. You can replace the 16px with $grid-unit-20 plainly.